scripts/lib.sh

```diff
@@ -159,9 +159,7 @@ function setup_fedimintd_env() {
   export FM_PASSWORD="pass$ID"
   export FM_FEDIMINTD_DATA_DIR="$FM_CFG_DIR/server-$ID"
 
-  # ensure datadir exists ... pipe to `true` because run_dkg() and setup_federation()
-  # both call this so the directory might already be here ...
-  mkdir $FM_FEDIMINTD_DATA_DIR || true
+  mkdir -p $FM_FEDIMINTD_DATA_DIR
 }
```

Comment:
This function has been [re-written in rust](https://github.com/fedimint/fedimint/blob/f56e63e70e43265f8e1d94d65aa0131904989a7a/fedimint-testing/src/bin/fixtures.rs#L308-L337) and can be deleted now

Reply:
https://github.com/fedimint/fedimint/pull/1916